index.html

```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Deadpool Connect 4</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="main.js"></script>
+</head>
+<body>
+
+<header class="header1">
+    <img class="headerImage" src="DPsprawl.png">
+</header>
+<main>
+    <div class="playerBoxBorder">
+        <div class="playerBox1 playerBox">
+            <div class="pieceDisplay1">
+                <img id="dpGamePiece" src="DPgamepiece.png">
+            </div>
+            <div class="player1Name"></div>
+            <div class="statsBox">
+                <div class="statsDisplay">
+                    <h3>Stats</h3>
+                    <h5>Games Played</h5>
+                    <h5>Attempts</h5>
+                </div>
+                <div class="forfeitButton">
+                    <button id="player1">
+                        Forfeit
+                    </button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="gameBoard">
+
+        <div id="01" class="row0">
+
+            <div class="space">
+                <div id="0" class="column0 column"></div>
```

Comment:
I think IDs that have just a number can lead to some big problems later on.  If you need to store a number, then store a custom attribute.

```<div column="0" row="1">```